docs/reference/rest-api/index.asciidoc

```diff
@@ -76,6 +77,7 @@ include::{es-repo-dir}/esql/esql-apis.asciidoc[]
 include::{es-repo-dir}/features/apis/features-apis.asciidoc[]
 include::{es-repo-dir}/fleet/index.asciidoc[]
 include::{es-repo-dir}/text-structure/apis/find-structure.asciidoc[leveloffset=+1]
+include::{es-repo-dir}/text-structure/apis/test-grok-pattern.asciidoc[]
```

Comment:
As the TOC is broken currently, we either need to add `leveloffset=+1` here or change every section level in the `test-grok-pattern.asciidoc` file one level deeper to make the section a subsection of REST APIs.

```suggestion
include::{es-repo-dir}/text-structure/apis/test-grok-pattern.asciidoc[leveloffset=+1]
```